core/src/main/res/values-v19/colors.xml

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
     <color name="selection_background_color_dark">#484B4D</color>
+    <color name="selection_background_color_trueblack">#000000</color>
```

Comment:
Is this necessary? I think if the value is not present in `values-v19`, it just uses the one from `values`

Reply:
Another bad mistake...